freqtrade/freqtradebot.py

```diff
@@ -2587,4 +2587,15 @@ def get_valid_price(self, custom_price: float, proposed_price: float) -> float:
         max_custom_price_allowed = proposed_price + (proposed_price * cust_p_max_dist_r)
 
         # Bracket between min_custom_price_allowed and max_custom_price_allowed
-        return max(min(valid_custom_price, max_custom_price_allowed), min_custom_price_allowed)
+        final_price = max(
+            min(valid_custom_price, max_custom_price_allowed), min_custom_price_allowed
+        )
+
+        # Log a warning if the custom price was adjusted by clamping.
+        if final_price != valid_custom_price:
+            logger.warning(
```

Comment:
i'm not sure if warning is the right level for this.

it's more an information - i'd keep warning for **actual** warnings.